.vscode/tasks.json

```diff
@@ -196,7 +196,12 @@
           "tell app \"Terminal\" to do script \"${workspaceFolder}/packages/cursorless-neovim/scripts/debug-neovim.sh ${workspaceFolder} development\" activate"
         ]
       },
-      // TODO: Add linux
+      "linux": {
+        "command": "scripts/linux-terminal.sh",
```

Comment:
should we move `linux-terminal.sh` to `packages/cursorless-neovim/scripts/` since it is neovim specific only? cc @pokey

Reply:
I think it makes sense to move it